azext_edge/tests/edge/support/create_bundle_int/helpers.py

```diff
@@ -22,6 +22,12 @@
 ]
 
 
+class NamespaceTuple(NamedTuple):
+    aio: str
+    usage_system: str
+    lmn_svclb: str
```

Comment:
lnm😣

Reply:
they should really just rename it

Reply:
they gonna disapear as a whole so let's just follow it now hahah